packages/desktop/views/login/views/update-stronghold/views/ChangePasswordView.svelte

```diff
@@ -0,0 +1,156 @@
+<script lang="ts">
+    import zxcvbn from 'zxcvbn'
+
+    import { OnboardingLayout } from '@components'
+    import { Animation, Button, PasswordInput, Text, TextHint } from '@ui'
+    import { ButtonSize, HTMLButtonType, TextType } from '@ui/enums'
+
+    import { localize } from '@core/i18n'
+    import { MAX_STRONGHOLD_PASSWORD_LENGTH } from '@core/profile'
+    import { changePasswordAndUnlockStronghold } from '@core/profile-manager'
+    import { updateStrongholdRouter } from '@core/router/subrouters/login'
+    import { PASSWORD_REASON_MAP } from '@core/stronghold'
+
+    export let actualPassword: string
+
+    let startOfPasswordChange: number
+
+    let password: string = ''
+    let passwordError: string = ''
+    let confirmPassword: string = ''
+    let confirmPasswordError: string = ''
+
+    let busy: boolean = false
+    let changeMessageLocale: string = ''
```

Comment:
this shouldnt be named `locale` because at the end it is used in `<Button><Button>`.